libmscore/mscore.cpp

```diff
@@ -375,6 +375,19 @@ void MScore::init()
                         exit(-1);
                   }
             }
+#endif
+// Workaround for QTBUG-73241 (solved in Qt 5.12.2) in Windows 10, see https://musescore.org/en/node/280244
+#if defined(Q_OS_WIN) && (QT_VERSION < QT_VERSION_CHECK(5, 12, 2))
+if (QOperatingSystemVersion::current().majorVersion() >= 10) {
+      QDir additionalFontsDir(QString("%1/AppData/Local/Microsoft/Windows/Fonts").arg(QStandardPaths::writableLocation(QStandardPaths::HomeLocation)));
```

Comment:
Technically, this should be:

    QDir additionalFontsDir(QString("%1/Microsoft/Windows/Fonts").arg(QStandardPaths::writableLocation(QStandardPaths::AppLocalDataLocation)));

Reply:
I thought so, but when looking at the docs https://doc.qt.io/qt-5.9/qstandardpaths.html#StandardLocation-enum it seems that QStandardPaths::AppLocalDataLocation returns a list of folders: "C:/Users/<USER>/AppData/Local/<APPNAME>", "C:/ProgramData/<APPNAME>", "<APPDIR>", "<APPDIR>/data", and I am not sure that the one I need is always the first one. I thought of using QStandardPaths::GenericConfigLocation, but it seems also to return a list of dirs: "C:/Users/<USER>/AppData/Local", "C:/ProgramData".

In principle, building the path as I am doing should work, and also in non-English systems.

Reply:
According to that documentation, that's just a list of possible values, and the function doesn't actually return a list:

> The following table gives examples of paths on different operating systems. The first path is the writable path (unless noted). Other, additional paths, if any, represent non-writable locations.

I did make a mistake and used `AppLocalDataLocation` instead of the correct `GenericDataLocation`. It will always return the correct value because the first one is the writable one, which you're explicitly requesting.

So the correct call should be (after also adding `const` for good measure):

    const QDir additionalFontsDir(QString("%1/Microsoft/Windows/Fonts").arg(QStandardPaths::writableLocation(QStandardPaths::GenericDataLocation)));

Reply:
Ok, thank you for the explanation.
I updated the PR.